src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -488,5 +492,13 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_command('list', 'list_Managed_private_endpoints')
         g.custom_command('delete', 'delete_Managed_private_endpoints', confirmation=True)
 
+    # Data Plane Commands --Artifacts Spark job definitions operations
+    with self.command_group('synapse spark-job-definition', synapse_spark_job_definition_sdk,
```

Comment:
Is this GA feature or preview feature? If it's preview, `is_preview=True` is needed.

Reply:
We do not need to add is_preview at the command level. The reason is that we set `is_preview=True` at the level of the entire module:
https://github.com/Azure/azure-cli/blob/dev/src/azurecli/azure/cli/command_modules/synapse/commands.py#L491-L492